File: clients/javascript/src/models/getStreamIn.ts
// this file is @generated





export interface GetStreamIn {
    name: string;
}

export const GetStreamInSerializer = {
    _fromJsonObject(object: any): GetStreamIn {
        return {
            name: object['name'],
            };
    },

    _toJsonObject(self: GetStreamIn): any {
        return {
            'name': self.name,
            };
    }
}

File: clients/rust/src/models/get_stream_in.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[non_exhaustive]
#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct GetStreamIn {
    pub name: String,
}

impl GetStreamIn {
    pub fn new(name: String) -> Self {
        Self { name }
    }
}
